apps/landing/src/styles/home.css

```
@@ -1307,15 +1309,87 @@ svg.inkline {
+   stay dark and sit on the cream as inset cards — the page already does this
+   with .screencap, and a bounded dark card reads as an embedded screen, which a
+   full-bleed dark band between two cream sections never did.
+   Only the bottom edge needs a seam, into .finale's slightly deeper cream —
+   exactly the boundary .testi used to own before this section existed.
+   Not a `.stage`: that forces min-height:100vh for the parallax sections. */
+.growth {
+  background: var(--paper);
+  color: var(--ink);
+  --line: var(--ink);
+  --seam-next: #ece2cf; /* .finale's background */
+  padding: 70px 22px 80px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)